pipeline/frontend/yaml/types/command.go

```diff
@@ -1,17 +1,18 @@
-package yaml
+package types
 
 import (
 	"errors"
 	"fmt"
 
 	"github.com/docker/docker/api/types/strslice"
+	// TODO: search for maintained
 	"github.com/flynn/go-shlex"
 )
 
 // Command represents a docker command, can be a string or an array of strings.
 type Command strslice.StrSlice
 
-// UnmarshalYAML implements the Unmarshaller interface.
+// UnmarshalYAML implements the Unmarshaler interface.
```

Comment:
I would keep the old spelling.
```suggestion
// UnmarshalYAML implements the Unmarshaller interface.
```